index.js

```diff
@@ -0,0 +1,286 @@
+var debug = require('debug')('loopback:filter');
+var geo = require('./lib/geo');
+
+module.exports = function filterNodes(nodes, filter) {
+  if (filter) {
```

Comment:
Ya I'll create an issue to talk about sanitization. I don't think we need it for `v0.1.0`.